packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -0,0 +1,134 @@
+import { captureFeedback } from '@sentry/core';
+import type { SendFeedbackParams } from '@sentry/types';
+import * as React from 'react';
+import type { KeyboardTypeOptions } from 'react-native';
+import { Alert, Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import {
+  CANCEL_BUTTON_LABEL,
+  EMAIL_ERROR,
+  EMAIL_LABEL,
+  EMAIL_PLACEHOLDER,
+  ERROR_TITLE,
+  FORM_ERROR,
+  FORM_TITLE,
+  IS_REQUIRED_LABEL,
+  MESSAGE_LABEL,
+  MESSAGE_PLACEHOLDER,
+  NAME_LABEL,
+  NAME_PLACEHOLDER,
+  SUBMIT_BUTTON_LABEL} from './constants';
+import defaultStyles from './FeedbackForm.styles';
+import type { FeedbackFormProps, FeedbackFormState } from './FeedbackForm.types';
+import LabelText from './LabelText';
+
+/**
+ * @beta
+ * Implements a feedback form screen that sends feedback to Sentry using Sentry.captureFeedback.
+ */
+export class FeedbackForm extends React.Component<FeedbackFormProps, FeedbackFormState> {
+  public constructor(props: FeedbackFormProps) {
+    super(props);
+    this.state = {
+      name: '',
+      email: '',
+      description: '',
+    };
+  }
+
+  public handleFeedbackSubmit: () => void = () => {
+    const { name, email, description } = this.state;
+    const { closeScreen, text } = this.props;
+
+    const trimmedName = name?.trim();
+    const trimmedEmail = email?.trim();
+    const trimmedDescription = description?.trim();
+
+    if (!trimmedName || !trimmedEmail || !trimmedDescription) {
+      const errorMessage = text?.formError || FORM_ERROR;
+      Alert.alert(text?.errorTitle || ERROR_TITLE, errorMessage);
+      return;
+    }
+
+    if (!this._isValidEmail(trimmedEmail)) {
+      const errorMessage = text?.emailError || EMAIL_ERROR;
+      Alert.alert(text?.errorTitle || ERROR_TITLE, errorMessage);
+      return;
+    }
+
+    const userFeedback: SendFeedbackParams = {
+      message: trimmedDescription,
+      name: trimmedName,
+      email: trimmedEmail,
+    };
+
+    captureFeedback(userFeedback);
+    closeScreen();
```

Comment:
Q: Are those two actions dependent on the order?
if not I'd concider closing the screen first and then capturing the feedback so users spends less time waiting for the feedback to be captured and spend more time on the app they were interacting
```suggestion
    closeScreen();
    captureFeedback(userFeedback);
```